feature-flag/src/main/kotlin/com/linecorp/android/featureflag/FeatureFlagTask.kt

```diff
@@ -125,9 +125,16 @@ abstract class FeatureFlagTask : DefaultTask() {
         buildEnvironment: BuildEnvironment
```

Comment:
This is out of the scope of this PR, but I've realized that we can call `VersionNumber.parse(applicationVersionName)` in `FeatureFlagPlugin.kt`

Reply:
No, we can't pass `VersionNumber` directly from `FeatureFlagPlugin`. We have to define a property of app version name for task cache key but `VersionNumber` doesn't implement `Serializable`.
https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:task_input_output_annotations

Reply:
Ah, you're right. Please ignore my previous comment.